### DISCUSSION TITLE
Cool project, seems to be abandoned

### DISCUSSION THREAD

**Discussion by AlexMikhalev** [Jasperav/Catalytic#30] [General]
Hello @Jasperav,
I liked your [blog post](https://www.scylladb.com/2022/02/15/introducing-catalytic-an-orm-designed-for-scylladb-and-cassandra-written-in-rust/) but the project seems to be abandoned. 
What are your plans for the future?

**Comment by Jasperav** [comment]
Hey @AlexMikhalev, thanks for you interest and for reading the blog. Why do you think the project is abandoned? It is using the very latest scylla rust driver and uses the latest Rust version. I even implemented the cargo workspace feature last month. I don't have anything to add to the current code at this moment, since it provides exactly what I need for my current projects, that is why I haven't committed in 2 weeks. 

If you have any feature requests, please report an issue :).
